src/Service/Lambda/CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 ## NOT RELEASED
 
+### Added
+
+- AWS api-change: Adds support for increased ephemeral storage (/tmp) up to 10GB for Lambda functions. Customers can now provision up to 10 GB of ephemeral storage per function instance, a 20x increase over the previous limit of 512 MB.
```

Comment:
only this line looks correct. the others either were already added in earlier releases or are not relevant to those actions/entities generated in this repo.